cpp/tests/io/csv_test.cpp

```diff
@@ -1577,26 +1581,21 @@ TEST_F(CsvReaderTest, MultiColumnWithWriter)
   cudf_io::csv_reader_options in_opts =
     cudf_io::csv_reader_options::builder(cudf_io::source_info{filepath})
       .header(-1)
-      .dtypes(std::vector<std::string>{"int8",
```

Comment:
don'e we need a test to handle strings as well

Reply:
The plan is to remove the strings C++ API, it will remain in Python.